### DISCUSSION TITLE
Tag for v109?

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#12971] [General]
Normally I point out that a released version hasn't been tagged, but for 109, things are a bit different.

My recently pulled local copy of the repo has a v109.0 tag dating back to 15th Dec 22 (e868d2e07c6e04b0fba2fab3783c8532aeab50b7), but on Github there is no such tag.

From past experience it seems that someone inadvertently created the 109 tag too soon and later deleted it, but due to the way git works, that version is forever stuck in my clone. Such that when the 'real 109' tag is created I'll keep getting a bunch of errors when updating tags that v109.0 already exists.

Please remember that Git Tags should not be considered as things that get replaced. Doing so really messes up extant clones.

I am assuming that the 109.0 release is based off the latest commit to the release/v109.0 branch (be71ea776c028ab7965aba982500429d8f38a896)??

**Comment by lmarceau** [comment]
Hi, thanks @gutley. Mistakes happens, and yes 109 was inadvertently "released" and the tag was deleted since it wasn't "released" yet. You assume correct, 109 is released with the latest commit on the release branch. Thanks.
